knowledge/craftable_spawneggs.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: craftable_spawners
+parent: Knowledge
+---
+ 
+# Using MoLang to create craftable spawn-eggs
+ 
+Minecraft Bedrock 1.16 introduced the ability to use MoLang expressions in the `data` field of recipe results. We can use the `query.get_actor_info_id` function to get the runtime integer ID of a given entity.
+
+
```

Comment:
Can you include an example for custom entities as well? Or at least a note that its possible?